project/eScavenge/models.py

```diff
@@ -236,6 +236,16 @@ def get_status(self, now, username):
                                    current_team.current_landmark+1, self.landmarks.all().count(),
                                    str(current_time_calc).split(".")[0],
                                    str(total_time + current_time_calc).split(".")[0])
+        if current_team.current_landmark == self.landmarks.all().count(): #When on final landmark, no need to add 1 to landmark index on print
```

Comment:
This seems wrong. Say there are 5 landmarks, if I'm on the final landmark my current_landmark index should be 4, we still need to add one on display.